src/tensors/indexmanipulations.jl

```diff
@@ -279,8 +279,8 @@ See also [`permute`](@ref), [`permute!`](@ref), [`add_braid!`](@ref), [`add_tran
                                           α::Number,
                                           β::Number,
                                           backend::AbstractBackend...) where {T,S,N₁,N₂}
```

Comment:
Not sure why all the type parameters are still here even after previous changes, but it seem they can be removed. Similarly for the braid and transpose methods below.